user_documentation_cf_cli_plugin.md

```diff
@@ -24,6 +26,9 @@ Subhankar Chattopadhyay
    1. [Listing all backups of a service-instance](#listing-all-backups-of-a-service-instance)
    1. [Starting a restore](#starting-a-restore)
    1. [Aborting a restore](#aborting-a-restore)
+   1. [Listing backups for deleted service instances](#listing-deleted-backups)
```

Comment:
Just to keep inline with above, 
"Listing all backups of a deleted service-instance"

Reply:
Also move the list backups above restore one, after [this](https://github.com/cloudfoundry-incubator/service-fabrik-cli-plugin/pull/2/files#diff-afbcd56c53b243872332d1755907d46dL24). Same below in the actual content as well, move after [this](https://github.com/cloudfoundry-incubator/service-fabrik-cli-plugin/pull/2/files#diff-afbcd56c53b243872332d1755907d46dL118).